python/jupyterlab_widgets/src/output.ts

```diff
@@ -114,7 +117,7 @@ export class OutputModel extends outputBase.OutputModel {
     }
   }
 
-  widget_manager: WidgetManager;
+  widget_manager: LabWidgetManager;
```

Comment:
Unfortunately changing from a more specialized to a more generic type here looks like a breaking change.

Reply:
> Thinking that another approach could be for those defining their own widget manager based on a `KernelWidgetManager` to also define their own `OutputModel` and register it

Linking to https://github.com/voila-dashboards/voila/pull/846/commits/38409998b5db79a6fb7bc78dba78325f2116b305 which implements this approach (may be temporary).

Reply:
> Unfortunately changing from a more specialized to a more generic type here looks like a breaking change.

Having breaking changes and therefore a major version bump on the lab manager isn't necessarily that problematic though.